ExampleMod/ExamplePlayer.cs

```diff
@@ -690,6 +690,77 @@ public class ExamplePlayer : ModPlayer
 			layers.Insert(0, MiscEffectsBack);
 			MiscEffects.visible = true;
 			layers.Add(MiscEffects);
+
+			/* This shows an example of how we can give the held sprite of an item a glowmask or animation. This example goes over drawing a mask for a specific item, but you may want to
+			 * expand your layer to cover multiple items, rather than adding in a new layer for every single item you want to create a mask for.
+			 * 
+			 * PlayerLayers have many other uses, basically anything you would want to visually create on a player, such as a custom accessory layer, holding a weapon, or anything else you can think of
+			 * that would involve drawing sprites on a player or their held items. these examples only serve to illustrate common requests and can be extrapolated into anything you want. DrawData's constructor has
+			 * the same set of params as SpriteBatch.Draw(), so you can get as creative as you want here.
+			 * 
+			 * Note that if you want to give your held item an animation, you should set that item's NoUseGraphic field to true so that the entire spritesheet for that item wont draw.
+			 */
+
+			//This layer is for our glowing sword example!
+			Action<PlayerDrawInfo> layerTarget = s => DrawSwordGlowmask(s); //the Action<T> of our layer. This is the delegate which will actually do the drawing of the layer.
+			PlayerLayer layer = new PlayerLayer("ExampleMod", "Sword Glowmask", layerTarget); //Instantiate a new instance of PlayerLayer to insert into the list
+			layers.Insert(layers.IndexOf(layers.FirstOrDefault(n => n.Name == "Arms")), layer); //Insert the layer at the appropriate index. 
```

Comment:
You need to add a nullcheck here for FirstOrDefault since it can return null, which will cause an exception. Modders can potentially remove this layer and cause this to happen. Also can't you pass the draw action directly? `new PlayerLayer("ExampleMod", "Sword Glowmask", DrawSwordGlowmask);` if so, that'd be a lot neater (direct reference to method is a lot cleaner here than a variable not being re-used)